src/tests/unit/tests/DetailsView/components/adhoc-tab-stops-test-view.test.tsx

```diff
@@ -12,6 +12,7 @@ import {
     VisualizationStoreData,
 } from 'common/types/store-data/visualization-store-data';
 import { VisualizationType } from 'common/types/visualization-type';
+import { DetailsViewActionMessageCreator } from 'DetailsView/actions/details-view-action-message-creator';
```

Comment:
Should the changes to this test (+ its snapshot) be reverted now that the changes to the component were reverted?

Reply:
Good catch!